models/reddit_ads.yml

```diff
@@ -154,12 +154,12 @@ models:
             - post_id
             - currency
     columns:
-      - name: date_day
-        description: '{{ doc("date_day") }}'
+      - name: date_day 
+        description: '{{ doc("date_day") }}' 
       - name: ad_name
         description: '{{ doc("ad_name") }}'
-      - name: ad_id
-        description: '{{ doc("ad_id") }}'
+      - name: ad_id 
+        description: '{{ doc("ad_id") }}' 
```

Comment:
Should we apply not_null tests to these fields?

Reply:
updaetd